java-frontend/src/main/java/org/sonar/java/se/xproc/MethodBehavior.java

```diff
@@ -20,24 +20,18 @@
 package org.sonar.java.se.xproc;
 
 import com.google.common.collect.ImmutableList;
-import org.sonar.java.resolve.JavaSymbol;
-import org.sonar.java.se.ExplodedGraph;
-import org.sonar.java.se.checks.SECheck;
-import org.sonar.java.se.constraint.ConstraintsByDomain;
-import org.sonar.java.se.symbolicvalues.SymbolicValue;
-import org.sonar.plugins.java.api.semantic.Symbol;
-import org.sonar.plugins.java.api.semantic.Type;
-
-import javax.annotation.Nullable;
-
 import java.util.ArrayList;
 import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Set;
 import java.util.stream.Stream;
+import org.sonar.java.se.ExplodedGraph;
+import org.sonar.java.se.checks.SECheck;
+import org.sonar.java.se.constraint.ConstraintsByDomain;
+import org.sonar.java.se.symbolicvalues.SymbolicValue;
+import org.sonar.plugins.java.api.semantic.Type;
 
 public class MethodBehavior {
-  private Symbol.MethodSymbol methodSymbol;
   private boolean varArgs;
   private boolean isStaticMethod;
```

Comment:
`staticMethodField` can also be removed, it is only passed around from CFG to `BytecodeEGWalker.startingStates` , but otherwise it's useless